ansible/environments/local/group_vars/all

```diff
@@ -32,3 +32,4 @@ kafka_topics_invoker_retentionBytes: 104857600
 kafka_topics_invoker_retentionMS: 300000
 
 env_hosts_dir: "{{ playbook_dir }}/environments/local"
+controller_protocol: https
```

Comment:
Is there a reson for not setting the `distributed` environment to https as well?
If there is no reason, can we make the default to https and leave these variables in the environements' files?